playwright.config.js

```diff
@@ -0,0 +1,22 @@
+import dotenv from 'dotenv'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+dotenv.config({
+    path: [
+        path.resolve(__dirname, '.env'),
+        path.resolve(__dirname, 'tests', 'playwright', '.env.testing')
+    ],
+    quiet: true
+})
```

Comment:
I'd some trouble with the order of loading locally, fixed that within: https://github.com/rapidez/account/pull/73, maybe also use that here to keep things the same.